client/signup/jetpack-connect/sso.jsx

```diff
@@ -44,18 +46,15 @@ import EmailVerificationGate from 'components/email-verification/email-verificat
  */
 const debug = debugModule( 'calypso:jetpack-connect:sso' );
 
-const JetpackSSOForm = React.createClass( {
-	displayName: 'JetpackSSOForm',
+class JetpackSSOForm extends Component {
 
```

Comment:
Although our linter stays silent on this one, let's remove the empty line in the beginning of the class.